src/geo/projection/globe.js

```diff
@@ -415,9 +414,8 @@ export class GlobeTile {
         this._brUp = latLngToECEF(br[0], br[1]);
         this._blUp = latLngToECEF(br[0], tl[1]);
 
-        const topEcefPerMeter = mercatorZfromAltitude(1, 0.0) * 2.0 * globeRefRadius * Math.PI;
-        const bottomEcefPerMeter = mercatorZfromAltitude(1, 0.0) * 2.0 * globeRefRadius * Math.PI;
-        //const pixelsPerMeter = mercatorZfromAltitude(1, 0.0) * (1 << tileID.canonical.z) * 512.0;
+        const topEcefPerMeter = EXTENT;
+        const bottomEcefPerMeter = EXTENT;
```

Comment:
Actually I'll just remove these values and use the constant directly to the scaling below.